content/en/platform/corda/5.0-alpha/deploying/prerequisites.md

```diff
@@ -8,24 +8,51 @@ menu:
     weight: 1000
 section_menu: corda-5-alpha
 ---
-<!--https://r3-cev.atlassian.net/browse/DOC-4188-->
-This section lists the third-party prerequisites for multi-worker cluster deployments. <!--For information about the prerequisites for local deployment with the [CorDapp Standard Development Environment (CSDE)](../cordapp-standard-development-environment/csde.html), see [Third-Party Prerequisites for the CSDE](../getting-started/prerequisites.html).-->
 
-## Software Prerequisites
+This section lists the third-party software prerequisites for multi-worker cluster deployments.
+The prerequisites for both the target environment and the environment from which you are deploying are listed.
+<!--For information about the prerequisites for local deployment with the [CorDapp Standard Development Environment (CSDE)](../cordapp-standard-development-environment/csde.html), see [Third-Party Prerequisites for the CSDE](../getting-started/prerequisites.html).-->
 
-Corda 5 cluster deployments have been tested with the following:
+## Local Environment
 
-| Software      | Version |
-| ----------- | ----------- |
-| Kubernetes (incl. kubectl)      |     |
-| Helm   |  |
+| Software | Version        |
+|----------|----------------|
+| kubectl  | 1.23           |
+| Helm     | 3.9.4 or newer |
+| Docker   |                |
```

Comment:
Docker version missing?

Reply:
We don't really have a requirement on a particular Docker version.